contributors/design-proposals/thirdpartyresources.md

```diff
@@ -0,0 +1,194 @@
+# ThirdPartyResources - 2
+
+## Background
+There are numerous, fundamental issues with TPRs today that we’ve tried to keep
```

Comment:
There are a number of important issues with the alpha version of ThirdPartyResources that we wish to address to move TPR to beta.  The list is tracked here, and also includes feedback from existing Kubernetes ThirdPartyResource users.  This proposal covers the steps we believe are necessary to move TPR to beta and to prevent future challenges in upgrading.

:)